src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -100,6 +109,17 @@ public HostStatus GetHostStatus()
             return status;
         }
 
+        [HttpPost]
+        [Route("admin/host/debug")]
+        public bool LaunchDebugger()
```

Comment:
Can we return an HttpResponseMessage instead here, so we can return 200 if successfull, or if not local, an appropriate error response?